sympy/printing/tests/test_repr.py

```diff
@@ -67,6 +67,11 @@ def test_Function():
     sT(sin, "sin")
 
 
+def test_Heaviside():
+    sT(Heaviside(x), "Heaviside(Symbol('x'))")
+    sT(Heaviside(x, 1), "Heaviside(Symbol('x'), Integer(1))")
```

Comment:
Maybe the `srepr` printer should not be changed...

Reply:
No, it should not, or else you also have to change lambdify again ;)

Reply:
In case you are looking for the documentation you have to fix, search for regular expression ´Heaviside\([^\(]+1/2\)´

Reply:
I think it's the StrPrinter method that affects the code printers but I fond a fix for that (see below).